packages/block-library/src/loginout/block.json

```diff
@@ -43,6 +43,18 @@
 				"fontSize": true
 			}
 		},
+		"__experimentalBorder": {
+			"radius": true,
+			"color": true,
+			"width": true,
+			"style": true,
+			"__experimentalDefaultControls": {
+				"radius": false,
+				"color": false,
+				"width": false,
+				"style": false
```

Comment:
I believe generally when the border supports have been adopted their controls are being displayed by default.

Reply:
Agreed.